validator/src/main.rs

```diff
@@ -83,6 +85,8 @@ enum Operation {
 
 const EXCLUDE_KEY: &str = "account-index-exclude-key";
 const INCLUDE_KEY: &str = "account-index-include-key";
+const DEFAULT_MIN_SNAPSHOT_DOWNLOAD_SPEED: f32 = 10485760_f32;
```

Comment:
Can we avoid a float here?  Also please document the units used

Reply:
Ack